spec/Dockerfile_spec.rb

```diff
@@ -18,7 +18,7 @@
 describe 'Dockerfile' do
   before(:all) do
     Docker.options[:read_timeout] = 900
-    @image = Docker::Image.build_from_dir('.')
+    @image = Docker::Image.get('sonatype/nexus3')
```

Comment:
I think we want to try to ensure that the image is the one we just built using a particular docker tag. If not we could be testing an older image or one pulled from docker hub.

Reply:
Replaced usage of the docker plugin with command line docker. The plugin is using image name rather than image id and there is no way to get image id from the plugin. Tried creating the image, then getting the image object from the plugin by image id. The plugin did not work properly from there as it was now using the image id for the image name internally.

(03e3fe1)